gpcontrib/gg_tables_tracking/sql/gg_tables_tracking_guc.sql

```diff
@@ -0,0 +1,226 @@
+-- start_ignore
+\! gpconfig -c shared_preload_libraries -v 'gg_tables_tracking'
+\! gpstop -raq -M fast
+\! gpconfig -c gg_tables_tracking.tracking_worker_naptime_sec -v '5'
+\! gpstop -u
+\c
+-- end_ignore
+-- start_matchsubs
+--
+-- m/ERROR:  \[gg_tables_tracking\] exceeded maximum number of tracked databases \(track_files\.c:\d+\)/
+-- s/\d+/XXX/g
+--
+-- end_matchsubs
+--start_ignore
+DROP DATABASE IF EXISTS tracking1;
+DROP DATABASE IF EXISTS tracking2;
+DROP DATABASE IF EXISTS tracking3;
+DROP DATABASE IF EXISTS tracking4;
+DROP DATABASE IF EXISTS tracking5;
+DROP DATABASE IF EXISTS tracking6;
+--end_ignore
+
+-- Test database registering GUC.
+CREATE DATABASE tracking1;
+\c tracking1;
+CREATE EXTENSION gg_tables_tracking;
+
+SHOW gg_tables_tracking.tracking_is_db_tracked;
+
+SELECT datname, setconfig FROM pg_db_role_setting JOIN pg_database ON
+setdatabase=oid WHERE datname=current_database();
+
+SELECT pg_sleep(current_setting('gg_tables_tracking.tracking_worker_naptime_sec')::int * 2);
```

Comment:
Maybe add something like `diskquota.wait_for_worker_new_epoch` to wait for functions to run in the background worker process?